src/components/ModelSelector.js

```diff
@@ -10,19 +8,17 @@ import {
   ToggleButton,
 } from "react-bootstrap";
 
-import Controls from "./Controls";
-import { useVolumeContext } from "../../context/volume-context";
-import { myChangeColorMapAction as changeColorMap } from "../../redux/AppActions";
-import { colorMaps } from "../../assets/config.json";
+import { useSelectorContext } from "../context/selector-context";
+import { myChangeColorMapAction as changeColorMap } from "../redux/AppActions";
+import { colorMaps } from "../assets/config.json";
 
-export default function ControlPanel(props) {
-  const [sidebarVisible, setSidebarVisible] = useState(false);
+export default function ModelSelector(props) {
+  const { sidebarVisible, setSidebarVisible } = props;
```

Comment:
can you do this destructuring in the function declaration instead? or is that not idiomatic?

Reply:
Ohhh you mean instead of `props`, yeah that totally works. Sorry I thought you mean where `useState` was called.

Tutorials usually show `props` in the declaration like I had originally but since it's only 1 hook being passed down I agree it's cleaner with the destructuring in the declaration